### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --hidden --format brief`

### CODE REVIEW

File: Kernel/Core/lib/mem/boxed.rs [hunk 1]
```diff
@@ -99,7 +99,7 @@ impl<T> ops::BoxPlace<T> for IntermediateBox<T> {
 				::memory::heap::ZERO_ALLOC as *mut u8
 			}
 			else {
-				// SAFE: Tell me again, why is allo unsafe?
+				// SAFE: Tell me again, why is allow unsafe?
```
Comment (thepowersgang) [inline Kernel/Core/lib/mem/boxed.rs]: `alloc`, not `allow` - Did you review this?

File: Kernel/Modules/block_cache/lib.rs [hunk 2]
```diff
@@ -90,7 +90,7 @@ impl CachedVolume
 	pub fn idx(&self) -> usize {
 		self.vh.idx()
 	}
-	/// Get the size fo a block on the underlying volume
+	/// Get the size for a block on the underlying volume
```
Comment (thepowersgang) [inline Kernel/Modules/block_cache/lib.rs]: `of` not `for` - Bug report on the typo fixing tool :)

File: Kernel/Modules/input_ps2/pl050.rs [hunk 3]
```diff
@@ -45,7 +45,7 @@ impl Port
 			};
 		log_debug!("p.base = {:?}", p.base);
 
-		// TODO: Unknown what this does, Acess did it.
+		// TODO: Unknown what this does, Access did it.
```
Comment (thepowersgang) [inline Kernel/Modules/input_ps2/pl050.rs]: While strictly speaking this is a "correct" fix... `Acess` is the actual name of the project being referenced.

File: Kernel/rundir/arm_bootloader/fdt-realview-pb-a8.dts [hunk 4]
```diff
@@ -321,7 +321,7 @@
 			address_type = "memory";
 			guest_physical_addr = <0x40000000>;
 			physical_size = <0x01000000>;
-			device_type = "alloced_rom";
+			device_type = "allocated_rom";
```
Comment (thepowersgang) [inline Kernel/rundir/arm_bootloader/fdt-realview-pb-a8.dts]: Probably not a correct fix, as this is an external API

File: Notes/AsyncTake3.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ Chained Operations
 E.g.
 - Fragmented file read
   - Just keep looping at an async level, easy.
-- Meatadata lookup (acceptable for that to be done in a worker or to be blocking)
+- Metaadata lookup (acceptable for that to be done in a worker or to be blocking)
```
Comment (thepowersgang) [inline Notes/AsyncTake3.md]: "Metadata" - Bug report on your fixing tool

File: Notes/TODO.txt [hunk 6]
```diff
@@ -23,7 +23,7 @@
 Named object handles (or just named VFS roots?)
 
 # Userland
-- AcessNative-esque test suite?
+- AccessNative-esque test suite?
```
Comment (thepowersgang) [inline Notes/TODO.txt]: See previous comments about "Acess" being the correct name

File: Notes/UserAPI-Networking.md [hunk 7]
```diff
@@ -21,7 +21,7 @@ Requirements
 
 - Socket Addressing:
  1. Assume IPv6/IPv4 only and use ::192.168.1.1 style encoding
- 2. Use the Acess model of type+address
+ 2. Use the Access model of type+address
```
Comment (thepowersgang) [inline Notes/UserAPI-Networking.md]: "Acess" again

File: Usermode/liballoc_system/lib.rs [hunk 8]
```diff
@@ -4,7 +4,7 @@
 #![crate_type="rlib"]
 #![crate_name="alloc_system"]
 #![feature(lang_items)]	// Allow definition of lang_items
-//#![feature(optin_builtin_traits)]	// For !Send
+//#![feature(option_builtin_traits)] // For !Send
```
Comment (thepowersgang) [inline Usermode/liballoc_system/lib.rs]: This is a rust feature name, so it was correct originally.

File: Usermode/loader/bin/src/elf/mod.rs [hunk 9]
```diff
@@ -470,15 +470,15 @@ impl PHEnt
 }
 struct PhEntIterator<'a, R: 'a + Read>
 {
-	file: &'a mut R,	// File is pre-seeked to the start of the PHENT list
+	file: &'a mut R,	// File is pre-sought to the start of the PHENT list
```
Comment (thepowersgang) [inline Usermode/loader/bin/src/elf/mod.rs]: "seeked" is correct in context, as it refers to the "seek" syscall

**Review by thepowersgang — CHANGES_REQUESTED** [review]
Blindly using a script is usually not a good idea, make sure to sanity check its output before committing.

**Review by thepowersgang — APPROVED** [review]

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**